Userland/Libraries/LibWeb/Layout/FormattingContext.cpp

```diff
@@ -842,7 +842,7 @@ void FormattingContext::compute_height_for_absolutely_positioned_non_replaced_el
     auto const& computed_min_height = box.computed_values().min_height();
     auto const& computed_max_height = box.computed_values().max_height();
 
-    if (!computed_max_height.is_none())
+    if (!computed_max_height.is_auto())
```

Comment:
`auto` is not a valid value for `max-height`: https://www.w3.org/TR/css-sizing-3/#max-size-properties

Reply:
Ok, Thanks